app/Http/Resources/ProductResource.php

```diff
@@ -35,13 +35,13 @@ public function index(Request $request): array
                 )
                 : [],
             'relevancy' => [
-                'title_position' => $this->resource->title_position, // @phpstan-ignore-line,
-                'title_relevancy' => $this->resource->title_relevancy, // @phpstan-ignore-line
-                'title_natural_relevancy' => $this->resource->title_natural_relevancy, // @phpstan-ignore-line
-                'title_words_relevancy' => $this->resource->title_words_relevancy, // @phpstan-ignore-line
-                'content_natural_relevancy' => $this->resource->content_natural_relevancy, // @phpstan-ignore-line
-                'content_relevancy' => $this->resource->content_relevancy, // @phpstan-ignore-line
-                'slug_length' => $this->resource->slug_length, // @phpstan-ignore-line
+                'title_position' => is_infinite($this->resource->title_position) || is_nan($this->resource->title_position) ? null : $this->resource->title_position, // @phpstan-ignore-line,
```

Comment:
Why `null` in some lines but `0` in other? I think that all of these should return a number?